vueui/src/components/view/turrets/turret-controls.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div>
+    <div class="item">
+      Behaviour controls are {{locked ? "locked" : "unlocked"}}.
+    </div>
+    <div v-if="access">
+      <div class="item">
```

Comment:
Uhm, there is undocumented `<vui-item>` component for such things. - https://github.com/Aurorastation/Aurora.3/blob/master/vueui/src/components/vui/item.vue
```vue
<vui-item label="Turret Status:" :balance="0.4">
    <vui-button :params="{'command' : 'enable', 'value' : 1}" :class="{'redButton': enabled}">Enabled</vui-button>
    <vui-button :params="{'command' : 'enable', 'value' : 0}" :class="{'selected': !enabled}">Disalbed</vui-button>
</vui-item>
```